docs/src/modules/components/MarkdownDocs.js

```diff
@@ -132,6 +139,7 @@ MarkdownDocs.contextTypes = {
   activePage: PropTypes.shape({
     pathname: PropTypes.string.isRequired,
   }).isRequired,
+  userLanguage: PropTypes.string,
```

Comment:
Should it be required?

Reply:
It should be fine - if it isn't provided (and it is), then it would be undefined, and the default "Edit this page" button would be rendered.